src/main/java/com/autotune/analyzer/recommendations/term/Terms.java

```diff
@@ -146,6 +145,8 @@ public static void setDurationBasedOnTerm(ContainerData containerDataKruizeObjec
             durationSummation = maxDurationInMinutes;
         }
         double durationSummationInHours = durationSummation / KruizeConstants.TimeConv.NO_OF_MINUTES_PER_HOUR;
+        durationSummationInHours = Double.parseDouble(String.format("%.2f", durationSummationInHours));
+        LOGGER.debug("durationSummationInHours = {}", durationSummationInHours);
```

Comment:
Can we remove this LOGGER, this is not useful as the info is available in the JSON anyway

Reply:
Done!